src/pages/FarmV3/FarmHeader.tsx

```diff
@@ -223,145 +174,63 @@ export const FarmHeader: FC = () => {
       ]
 
   return (
-    <>
-      <HEADER_WRAPPER>
-        {poolSelectionModal && (
-          <ChoosePool poolSelectionModal={poolSelectionModal} setPoolSelectionModal={setPoolSelectionModal} />
-        )}
-        <div tw="flex flex-col cursor-pointer w-15 mr-2.5 sm:w-12.5 sm:h-15">
-          <div
-            css={[tw`duration-500`, range === 0 ? tw`mt-0` : range === 1 ? tw`mt-5` : tw`mt-10 sm:mt-[35px]`]}
-            tw="h-5 bg-gradient-1 w-15 absolute rounded-[100px] sm:w-12.5 sm:h-[25px]"
-          ></div>
-          <h4
-            css={[range === 0 ? tw`!text-white` : tw`text-grey-1`]}
-            tw="h-5 duration-500 flex items-center z-[1] justify-center font-bold
-                            w-15 text-tiny sm:w-12.5 sm:h-[25px] sm:mb-2.5"
-            onClick={() => setRange(0)}
-          >
-            24H
-          </h4>
-          {!checkMobile() && (
-            <h4
-              css={[range === 1 ? tw`!text-white` : tw`text-grey-1`]}
-              tw="h-5 duration-500 flex items-center z-[1] justify-center
-                  font-bold w-15 text-tiny"
-              onClick={() => setRange(1)}
-            >
-              7D
-            </h4>
-          )}
-          <h4
-            css={[range === 2 ? tw`!text-white` : tw`text-grey-1`]}
-            tw="h-5 flex items-center justify-center z-[1] font-bold
-                            w-15 text-tiny sm:w-12.5 sm:h-[25px]"
-            onClick={() => setRange(2)}
+    <div
+      className={`flex flex-row relative items-center no-scrollbar gap-2.5 overflow-x-scroll
+      p-5 pt-3.75 sm:pl-2.5 sm:pr-0 pb-0
+      `}
+    >
+      {poolSelectionModal && (
+        <ChoosePool poolSelectionModal={poolSelectionModal} setPoolSelectionModal={setPoolSelectionModal} />
+      )}
+      <RadioOptionGroup
+        optionSize={isMobile ? 'xl' : 'sm'}
+        defaultValue={'24h'}
+        orientation={'vertical'}
+        className={'gap-0'}
+        options={[
```

Comment:
Can these options be memoized?

Reply:
Good catch, rapid development meant something like this slipped through.. ty